docs/src/matrix.md

```diff
@@ -215,6 +215,10 @@ length(::MatElem)
 isempty(::MatElem)
 ```
 
+```@docs
+zero_matrix(::Ring, ::Int, ::Int)
```

Comment:
This function is part of the matrix interface and must be implemented by all matrix types. It isn't functionality provided generically for all matrix types by AbstractAlgebra. Therefore documenting it here is incorrect.

Reply:
Okay, I will remove it then.